interface/forms/newpatient/new.php

```diff
@@ -22,6 +22,7 @@
     ($tmp['squad'] && ! AclMain::aclCheckCore('squads', $tmp['squad'])) ||
     !AclMain::aclCheckForm('newpatient', '', array('write', 'addonly'))
 ) {
+    // TODO: why is this reversed?
     echo "<body>\n<html>\n";
```

Comment:
agree that this is very odd :)

Reply:
I left this alone... but I can change it if needed.

Reply:
no need to change. will preserve for historical purposes :)